source/CIPlanetNode.h

```diff
@@ -96,7 +107,7 @@ class PlanetNode : public cugl::scene2::AnimationNode {
                     setScale(_coreScale);
                 }
                 node->innerRing->setScale(_layerScale/_coreScale);
-                node->outerRing->setScale(_layerScale/_coreScale);
+                node->outerRing->setScale(_layerScale/(1.2*_coreScale));
```

Comment:
Can we turn the 1.2 into a #define at the top of the file? Also, I think it would make a bit more sense to multiply _layerScale by the constant (so turn it into 1/1.2 = .833).

Another thing - do you want the outer ring to be that far away from the inner ring? There is quite the gap currently and most other visuals of the planet have them a bit closer.

Reply:
Right now it's hard to see the animation if the outer ring is too close to the inner ring.